app/javascript/packs/intl-tel-input.js

```diff
@@ -2,25 +2,30 @@ import { loadPolyfills } from '@18f/identity-polyfill';
 import 'intl-tel-input/build/js/utils.js';
 import * as intlTelInput from 'intl-tel-input/build/js/intlTelInput';
 
-function intlTelInputNormalize() {
+/**
+ * @param {HTMLDivElement} iti
+ */
+function intlTelInputNormalize(iti) {
   // remove duplicate items in the country list
-  const dupUsOption = document.querySelectorAll('#country-listbox #iti-item-us')[1];
-  if (dupUsOption) {
-    /** @type {HTMLElement} */ (dupUsOption.parentNode).removeChild(dupUsOption);
-  }
-  const dupCanOption = document.querySelectorAll('#country-listbox #iti-item-ca')[1];
-  if (dupCanOption) {
-    /** @type {HTMLElement} */ (dupCanOption.parentNode).removeChild(dupCanOption);
-  }
+  /** @type {NodeListOf<HTMLLIElement>} */
+  const preferred = iti.querySelectorAll('.iti__preferred');
+  preferred.forEach((listItem) => {
+    const { countryCode } = listItem.dataset;
+    /** @type {NodeListOf<HTMLLIElement>} */
+    const duplicates = iti.querySelectorAll(`.iti__standard[data-country-code="${countryCode}"]`);
+    duplicates.forEach((duplicateListItem) => {
+      duplicateListItem.parentNode?.removeChild(duplicateListItem);
```

Comment:
what are the cases when a list item has a null parent node? since we're querying from `iti` wouldn't every element have some parent?

Reply:
> what are the cases when a list item has a null parent node? since we're querying from `iti` wouldn't every element have some parent?

In most practical cases, never. Especially here since we're running `querySelectorAll` from a starting element, so at the very least that element would be the parent. I think TypeScript defaults to a generic `Element|null` type for the parent, since it can be a common case to "walk up the tree" using `null` as a condition to stop (e.g. [`Element#closest` polyfill](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest#Polyfill)).

Another option here is to instruct TypeScript to treat it as though it exists:

```suggestion
      /** @type {HTMLULElement} */ (duplicateListItem.parentNode).removeChild(duplicateListItem);
```

Reply:
Oh yeah I didn't realize it was part of the method definition --- seems easier to just leave the null-safe navigator in there

Reply:
> Oh yeah I didn't realize it was part of the method definition --- seems easier to just leave the null-safe navigator in there

Yeah, the main downside I see is that in practical terms, the transpiled output is quite verbose ([example](https://babeljs.io/repl#?browsers=ie%2011&build=&builtIns=false&spec=false&loose=false&code_lz=PYBwLglsB2CGA2B-AdAYwBawtbBzA3EA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=env&prettier=false&targets=&version=7.12.12&externalPlugins=)), though this feels pretty incidental and eventually temporary.